packages/bpk-component-inset-banner/src/BpkInsetBannerV2/BpkInsetBannerSponsored.tsx

```diff
@@ -86,7 +85,6 @@ const BpkInsetBannerSponsored = ({
             }}
           >
             <button
-              aria-label={callToAction?.buttonA11yLabel}
```

Comment:
Hey @hir06 thanks for raising the PR, can you explain more the duplicated places? I didn't quite get the context.